macros/cross-adapter-modeling/sessionization/segment_web_page_views__sessionized.sql

```diff
@@ -133,7 +133,14 @@ session_ids as (
         
         {{dbt_utils.star(ref('segment_web_page_views'))}},
         page_view_number,
-        {{dbt_utils.surrogate_key('anonymous_id', 'session_number')}} as session_id
+        {{dbt_utils.surrogate_key('anonymous_id', 'session_number')}} as session_id,
+        case
```

Comment:
The whole point of this model (as it talks about in its docs and comments) is to create a session_id, and this logic isn't related to that. This should be in `snowplow_web_page_views`, right alongside the `device` field you just created.